### PR TITLE
OUTPOST-65 bulk import custom fields

### PR DESCRIPTION
**Pull request by apricot13**
- [x] documentation
- [x] import script
- [x] tests

### CODE REVIEW

**Comment by em-cd** [comment]
@apricot13 I've added tests and made some changes to the import script - I think this is ready! Have a look over it and see what you think 😄

**Comment by em-cd** [comment]
I have ruined the code coverage by tweaking some of the codecov settings, but that's okay - I'd rather we had bad but accurate coverage and work to improve it, than bad coverage and not know about it!

File: lib/tasks/data_import/custom_fields/custom_fields.rake [hunk 1]
```diff
@@ -0,0 +1,139 @@
+namespace :import do
+  desc 'Import custom field from csv file to database. By default, looks for a file at lib/seeds/custom-fields.csv, or you can pass in a custom file path as an argument'
```
Comment (apricot13) [inline lib/tasks/data_import/custom_fields/custom_fields.rake:2]: oooh neat!!

File: lib/tasks/data_import/custom_fields/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Number
+Checkbox
+Select
+Date
+```
+
+
+# Running the import
+
+Running `rake import:custom_fields` will import the data from the custom-fields.csv file.
```
Comment (apricot13) [inline lib/tasks/data_import/custom_fields/README.md:62]: could you add an example of the arguments field in here?

does it take http:// or only files on the server etc

**Review by apricot13 — COMMENTED** [review]
wont let me approve my own PR!  👍  though!

File: lib/tasks/data_import/custom_fields/README.md (hunk 2 above)
Comment (em-cd) [reply lib/tasks/data_import/custom_fields/README.md:62]: Yes! Good point :)

File: lib/tasks/data_import/custom_fields/custom_fields.rake (hunk 1 above)
Comment (em-cd) [reply lib/tasks/data_import/custom_fields/custom_fields.rake:2]: only really added that to make testing easier but it could come in handy! 😄

**Pull request merged**